### PR TITLE
Add tidy automation

### PR DESCRIPTION
**Pull request by anssiko**

### CODE REVIEW

File: .github/workflows/tidy.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      - main
+
+jobs:
+  tidy:
+    name: Tidy up
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: brew install tidy-html5
+      - run: tidy -config tidyconfig.txt -o index.html index.html
```
Comment (mfoltzgoogle) [inline .github/workflows/tidy.yml:15]: The config file is at `build/tidyconfig.txt` - does this need an update?

Comment (anssiko) [inline .github/workflows/tidy.yml:15]:
```suggestion
      - run: tidy -config build/tidyconfig.txt -o index.html index.html
```

Comment (anssiko) [reply .github/workflows/tidy.yml:15]: Good catch.

**Review by mfoltzgoogle — APPROVED** [review]

**Review by xfq — APPROVED** [review]
LGTM. Thank you!

**Comment by anssiko** [comment]
@mfoltzgoogle this should automatically run tidy as needed -- we can consider rolling this automation to the WG's specs too if this works as intended.

**Pull request merged**